src/main/java/org/concordion/internal/ConcordionBuilder.java

```diff
@@ -21,13 +21,12 @@
 import org.concordion.internal.extension.ExtensionChecker;
 import org.concordion.internal.listener.*;
 import org.concordion.internal.parser.markdown.MarkdownConverter;
-import org.concordion.internal.util.Announcer;
 import org.concordion.internal.util.Check;
 import org.concordion.internal.util.IOUtil;
 
 public class ConcordionBuilder implements ConcordionExtender {
 
-    private Announcer<ConcordionBuildListener> listeners = Announcer.to(ConcordionBuildListener.class);
+    private List<ConcordionBuildListener> listeners = Collections.synchronizedList(new ArrayList<ConcordionBuildListener>());
```

Comment:
I don't think we should be adding synchonization as part of this PR. 
The listeners weren't thread-safe, and there's no need to add it now. Even with the parallel runner (for running specs in parallel), each thread has its own isolated set of listeners, built by ConcordionBuilder for the spec that the thread is processing.
Wider changes would be needed if we want to run examples in parallel, since the underlying XOM document is not thread-safe.